### PR TITLE
Leading slashes mqtt topics

### PR DESCRIPTION
**Pull request by jmezzera**
Same as https://github.com/telefonicaid/iotagent-ul/pull/429

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -99,7 +99,12 @@ config.mqtt = {
     /**
      * Time to keep connection open between client and MQTT broker (default is 0 seconds)
      */
-    keepalive: 0
+    keepalive: 0,
+
+    /**
+     * Whether to use slashes at the beginning of topic when sending or not
+     */
+    avoidLeadingSlashes: false
```
Comment (fgalan) [inline config.js:107]:
```suggestion
    avoidLeadingSlash: false
```

(as in the IOTA-UL PR)

Once applied, I understand this change would need fixes in some other places.

Comment (fgalan) [reply config.js:107]: Or maybe it's a typo just in this place... (other parts of the PR use the singular form)

Comment (jmezzera) [reply config.js]: My bad. Definitely a typo. Thanks btw

Comment (fgalan) [reply config.js]: Fixed in df397c4

**Pull request merged**